src/DynamoUtilities/Utils.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace DynamoUtilities
+{
+    public class Utils
+    {
+        public static readonly string GeometryFactoryAssembly = "LibG.ProtoInterface.dll";
+        public static readonly string PreloaderAssembly = "LibG.AsmPreloader.Managed.dll";
+        public static readonly string PreloaderClassName = "Autodesk.LibG.AsmPreloader";
+        public static readonly string PreloaderMethodName = "PreloadAsmLibraries";
```

Comment:
These aren't really set in stone forever.  You could probably move these out to a configuration file.